addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -45,7 +45,29 @@
 import org.zaproxy.zap.model.Vulnerabilities;
 import org.zaproxy.zap.model.Vulnerability;
 
-/** a scanner that looks for Path Traversal vulnerabilities */
+/**
+ * A scanner that looks for Path Traversal vulnerabilities (see <a
+ * href="https://owasp.org/www-community/attacks/Path_Traversal">OWASP Path Traversal</a>).
+ *
+ * <p>The following techniques are performed (depending on the selected attack strength) both with
+ * unix and windows specific paths and separators:
+ *
+ * <ul>
+ *   <li>Absolute Path Traversal
+ *   <li>Directory climbing (../)
+ *   <li>Using file URI scheme
+ *   <li>Using null byte injection
+ * </ul>
```

Comment:
This doesn't seem necessary. Doesn't really align with the 5 checks below, and they're already well commented.

Reply:
Regarding most of the other scan rules, this really looks not necessary - so fixed